src/java.desktop/share/classes/com/sun/java/swing/plaf/gtk/GTKFileChooserUI.java

```diff
@@ -564,6 +564,7 @@ public void installComponents(JFileChooser fc) {
 
         fc.setLayout(new BorderLayout());
         fc.setAlignmentX(JComponent.CENTER_ALIGNMENT);
+        fc.setMultiSelectionEnabled(true);
```

Comment:
Please double check that this will not override what the user set

Reply:
It will not override the user choice. If user has set `multiselection `to `false`, FileChooser won't allow to select all files or folders.

Reply:
@mrserb Please have a look on latest changes.

Reply:
Its better to add a comment which mentions why we enable MultiSelection by default in GTK LAF compared to other LAF's, since we are trying to match platform behaviour.

Reply:
Comment added.